proposals/revised-governance.md

```diff
@@ -0,0 +1,227 @@
+   up to each working group to decide on its own process for exercising that
+   vote.
+2. An Admin team would be created that would be charged with handling the
+   more mundane administrative tasks currently handled by the TSC (budget
+   allocations, github org management, handling confidential information).
+   Tasks like administering the travel fund could be handed off to the
+   Community Committee. This admin team would not be a voting body.
+3. There would be only one TSC meeting per month.
+   * The meeting may be in person or fully remote.
+   * The meeting would consist of a public portion only that includes a
+     stand up of *working group* activities and issues, and discussion of
+     escalated issues.
+   * Any working group member may attend.
+   * Anyone who is not a working group member may attend on invitation from a
+     working group member.
+4. The TSC Chair and TSC Director positions remain with an annual election.
+   * Any working group member is eligible to run for either seat and all working
+     group members would vote.
+   * The TSC Chair organizes the agenda for, and conducts, the monthly meeting.
+   * The TSC Director sits on the Node.js Board of Directors and chairs the
+     Admin team.
+5. Any working group member may request to participate in the Admin team, and
+   the Admin team may have a process for adding non-working group members.
+   All Admin team members are subject to consensus or vote by the working
+   groups. 
+6. The CTC does not change with the exception that it does not charter its own
+   working groups.
+7. Working groups would be chartered through consensus of all the working
+   groups, including the CTC. The CTC would not have sole ability to override
+   a working group decision. If the CTC objected to a decision made by 
+   chartered working group, the final decision would be made through consensus
+   or vote of all the working groups. 
+
+## Questions
+
+### Why not simply fold the TSC and CTC back together?
+
+For the simple reason that doing so does not address the cause of the
+disfunction.
+
+Also, the reason the TSC and CTC were split in the first place was because the
+overwhelming majority of CTC members do not want to deal with the administrative
+tasks delegated to the TSC, and I imagine that is still the case. We would still
+need an Admin team to handle those tasks.
+
+### Why working group votes?
+
+Currently, the TSC is the final arbiter on contentious decisions. If the CTC
+made a decision that a collaborator disagreed with, the issue can be flagged
+for TSC review and decision. While this *hardly ever happens*, it is an
+important step. The working group votes would only exist to preserve the
+ability to escalate issues in the exremely rare case it becommes necessary.
+But, rather than having a standing body of individuals decide the matter, it
+goes to a consensus of the working groups instead. It should be exceedingly
+rare for this to actually happen tho.
```

Comment:
Has it ever happened were the TSC was asked to review and /or overrule the CTC ?  I cant' remember this happening but may have overlooked something.